src/js/hca/components/veteran-information/VeteranAddressSection.jsx

```diff
@@ -10,19 +10,6 @@ import { veteranUpdateField } from '../../actions';
  * `reviewSection` - Boolean. Hides components that are only needed for ReviewAndSubmitSection.
  */
 class VeteranAddressSection extends React.Component {
-  constructor() {
```

Comment:
Yes, I just noticed this was supposed to be removed and figured might as well do it now. The instructions are no longer relevant because we have a separate country field now, and emails are in a different page so we don't need to validate that here.